px-datetime-entry-cell.html

```diff
@@ -246,12 +246,13 @@
       }
       ctx.font = fontSize + " " + fontFamily;
       if(this.$$('.datetime-entry-input').value) {
-        length = ctx.measureText(this.$$('.datetime-entry-input').value.toUpperCase()).width + 1;
+        length = ctx.measureText(this.$$('.datetime-entry-input').value.toUpperCase()).width;
```

Comment:
this should not have been changed, please revert. 

The spacing is accurate at 2, but much too spread out at 1.